lib/iex/lib/iex/evaluator.ex

```diff
@@ -254,6 +260,23 @@ defmodule IEx.Evaluator do
     :elixir_errors.parse_error(line, "iex", error, token)
   end
 
+  defp check_result(result, code) when is_list(result) do
+    if Enum.all?(result, &is_integer(&1)) do
+      input =
+        code
+        |> String.Chars.to_string
+        |> String.replace(" ", "")
+
+      string_result = String.Chars.to_string(result)
```

Comment:
I am not sure about how to convert result into string.
If we print in iex `[97,98]` `result` variable would be `'ab'` which is converted to `"ab"`. This is fine.
But if we print `[97,1]` `result` would be `[97,1]` and it would be converted to `<<97, 1>>`. It causes `String.contains?` to return `false` and we got 'learn more' message here.

Also how should we deal with `[97, ?b]`?